src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -543,6 +543,10 @@ In order to debug this project, add an executable project to this solution which
   <data name="FUTD_AccelerationDisabledCopyItemsIncomplete" xml:space="preserve">
     <value>Build acceleration is not available for this project because not all transitively referenced projects have provided acceleration data.</value>
   </data>
+  <data name="FUTD_AccelerationDisabledDuplicateCopyItemsIncomplete_1" xml:space="preserve">
+    <value>Build acceleration is not available for this project because it copies duplicate files to the output directory: {0}</value>
+    <comment>{0} is a list of one or more relative file paths.</comment>
```

Comment:
How are these file paths separated?

Reply:
They are single-quoted and comma-separated, which matches how we log other paths in the FUTDC logs.